onnxscript/function_libs/torch_lib/ops/nn.py

```diff
@@ -711,14 +711,16 @@ def _adjust_attributes_of_max_pool(
     else:
         kernel_shape = kernel_size
 
+    # expand_size is the dimension of pooling kernel,
+    # ONNX needs begin and end padding so we need to double the padding
     if isinstance(padding, int):
         pads = [padding] * expand_size * 2
     elif len(padding) == 1:
         pads = padding * expand_size * 2
     elif len(padding) == 2:
         pads = padding * expand_size
-    else:
-        pads = padding
```

Comment:
Not even sure if this is a scenario that we should keep.

Reply:
This branch is used to handle the case that user has provided the correct value of padding, in this example it might be (1, 1, 1, 1, 1, 1), so we don't need to do the expand work.

Reply:
Done